.github/workflows/deployment.yml

```diff
@@ -49,6 +49,9 @@ jobs:
       - name: Increment Android Version
         run: cd android && fastlane bump version:${{ steps.tagName.outputs.version }} && cd ..
 
+      - name: Create ENV File
+        run: echo "BASE_API_URL=${{ secrets.BASE_API_URL }}\nAPI_AUTH_TOKEN=${{ secrets.API_AUTH_TOKEN }}\nSENTRY_DTN=${{ secrets.SENTRY_DTN }}" > .env
```

Comment:
Is this something that needs documenting in the README’s setup instructions?

Reply:
I think I'll just add a default one for local builds, I meant to anyway.